netmanager/src/views/pages/Download/Download.js

```diff
@@ -92,6 +104,58 @@ const Download = (props) => {
     { value: "usual", label: "Usual" },
   ];
 
+  const listOfDeviceNumbers = [
```

Comment:
@ja-odur is it okay to leave the device details exposed like this?

Reply:
Cant these values be loaded through an endpoint? What happens when they change?

Reply:
Good question. The assumption is that these are the only devices being used for now and that since they are few and the feature is needed asap, hard coding them was the quickest route of implementation. 
You can find the conversation in the `platform` channel on slack